File: Selection_Sort/JAVA/Selection.java
public class Selection extends Sorting {
	public void sort(int array []) {
		int arr = array.length;
		for (int i = 0; i < arr - 1; i++) {
			int min_index = i;
			for (int j = i + 1; j < arr; j++) {
				if (array[j] < array[min_index]) {
					min_index = j;
				}
			}

			int tmp = array[min_index];
			array[min_index] = array[i];
			array[i] = tmp;
		}
	}
}

File: Selection_Sort/Python/Selection.py
def selection(array):
	arr = len(array)
	
	for i in range(arr):
		min_index = i
		for j in range(i + 1, arr):
			if array[min_index] > array[j]:
				min_index = j
		array[i], array[min_index] = array[min_index], array[i]
		
def print(array):
	arr = len(array)
	
	print("[")
	for i in range(arr):
		if i == arr - 1:
			print(array[i] + "]")
		else:
			print(array[i] + ", ")	